### ISSUE TITLE
Animation in README and on RTD is inaccessible

### ISSUE THREAD

**Issue by nkantar** [pypa/pipenv#3666]
### Issue description

The animation referenced in the README and on Read the Docs currently doesn't show up.

### Expected result

Seeing the animation.

### Actual result

The response from S3 is as follows:

```xml
<Error>
  <Code>AllAccessDisabled</Code>
  <Message>All access to this object has been disabled</Message>
  <RequestId>BB41E4330F22C8C6</RequestId>
  <HostId>7Nm9ixlUsFQEkIJJcU+jdU/6m47oM5m3/SXvB/+hNSTlJsd4AdcaPSl0FLWB/aFHgMPQJrvsJPU=</HostId>
</Error>
```

### Steps to replicate

1. Go to the [main repo page](https://github.com/pypa/pipenv) or [docs homepage](https://pipenv.readthedocs.io/en/latest/)
2. Scroll to where the animation is suppose to be
3. Observe the image not loading

### Screenshots

<img width="1516" alt="Screen Shot 2019-04-01 at 7 52 28 PM" src="https://user-images.githubusercontent.com/568543/55372753-b2619a00-54b7-11e9-825d-764640160042.png">
<img width="1516" alt="Screen Shot 2019-04-01 at 7 52 17 PM" src="https://user-images.githubusercontent.com/568543/55372752-b2619a00-54b7-11e9-9318-cc91ef9d1b63.png">

**Comment by frostming** [comment]
On my side, the Github caches the GIF in `githubusercontent` so it can
display normally, but the GIF is broken on documentation.

Thanks

>

**Comment by snow8gaia** [comment]
MS Edge-

window-
works on main repo page
broken on docs homepage

Inprivate window-
works on main repo page
broken on docs homepage

--------

Bromite(https://www.bromite.org/)-
[after delete `all cached images and tabs`]
tabs-
works on main repo page
broken on docs homepage

incognito tabs-
works on main repo page
broken on docs homepage

-------
 [main repo page](https://github.com/pypa/pipenv) or [docs homepage](https://github.com/pypa/pipenv)



<details>

<summary> Another issue?</summary>

<p>
testimonial is still visible. on the 8 instances above. https://github.com/pypa/pipenv/pull/3536/files
2 Apr 2019, 5:15 PM[`Asia/Kolkata`]

hope it helps.
</p>

</details>

[editiing]

**Comment by tuanluu-agilityio** [comment]
Anyone can help this issue? I can't see gif file on homepage.

**Comment by nkantar** [comment]
This issue has the "help wanted" label, but it seems that the error is tied to the AWS account hosting the image. I'd be happy to help if someone could provide guidance on acceptable alternative image hosting (perhaps GitHub via `githubusercontent`?).

**Comment by nachiketdhamankar** [comment]
Any update on this?
I noticed the issue too.
I get `Invalid upstream response (403)` for the 'image' in the [README.md](https://github.com/pypa/pipenv/blob/master/README.md)

**Comment by frostming** [comment]
Closed via #3850
